checks/checks.go

```diff
@@ -88,7 +88,8 @@ func runHTTPRequest(
 
 	resp, err := client.Do(req)
 	if err != nil {
-		result = api.HTTPRequestResult{Err: "Failed to fetch"}
+		errString := fmt.Sprintf("Failed to fetch - url: %s, error: %s", completeURL, err.Error())
```

Comment:
```suggestion
		errString := fmt.Sprintf("Failed to fetch - error: %s", err.Error())
```
I think showing the url might be overkill since the error we want to show to the user will be shown in the error string.

**current**
`Err: Failed to fetch - url: http://localhost:8091/, error: Get "http://localhost:8091/": dial tcp [::1]:8091: connect: connection refused`

**edited**
`Err: Failed to fetch - error: Get "http://localhost:8091/": dial tcp [::1]:8091: connect: connection refused`